pkgs/build-support/make-desktopitem/default.nix

```diff
@@ -12,6 +12,8 @@
 , mimeType ? null
```

Comment:
this probably is the reason why 553 packages get rebuild. we don't want to break any of them just before the release

please make this PR against staging branch

`90secondportraits abuse adapta-gtk-theme aesop agi aisleriot aj-snapshot akira alloy4 alloy5 alloy5 almanah altair-v4.0.2 android-studio-beta android-studio-canary android-studio-dev android-studio-stable anki anydesk apache-directory-studio apostrophe apostrophe appcenter appimage-run appimage-run-tests appimagekit-unstable Apple.Music assaultcube-unstable audiality2 authenticator avocode banking baobab basex beneath-a-steel-sky betaflight-configurator betterdiscord-installer bitwarden bluefish bookworm bottles brogue broken-sword caja-dropbox calls caprice32 carla ccemux celluloid celluloid chain-desktop-wallet CharacterCompressor charles charles chrome-gnome-shell chrysalis clion clipgrab clonehero code codium CompBus constant-detune-chorus contrast cozy csound-qt darktable datagrip dbeaver deltachat-electron desktop-file-utils (0.24 → 0.26) desktop_file_utils (0.24 → 0.26) desmume devdocs-desktop dirdiff discord discord-canary discord-ptb discover discover discover discover diylc dolphin-emu domination dosbox drascula-the-vampire-strikes-back drawing dreamweb dropbox dropbox-cli duckmarines durandal eagle eclipse-committers eclipse-cpp eclipse-java eclipse-jee eclipse-mat eclipse-modeling eclipse-platform eclipse-rcp eclipse-scala-sdk eclipse-sdk eduke32 electronplayer element-desktop elementary-calculator elementary-calendar elementary-camera elementary-capnet-assist elementary-code elementary-dpms-helper elementary-files elementary-greeter elementary-gsettings-desktop-schemas elementary-music elementary-photos elementary-planner elementary-screenshot-tool elementary-session-settings-unstable elementary-shortcut-overlay elementary-terminal elementary-videos envelope eolie ephemeral epiphany evilpixie extra-elementary-contracts factorio-alpha factorio-alpha factorio-demo faust2jack-unstable faust2jaqt-unstable faustCompressors-v1.2 feedreader ffado ffado file-roller firefox firefox firefox firefox firefox firefox firefox-beta-bin firefox-bin firefox-devedition-bin fish-fillets-ng flatpak flatpak-builder flexoptix-app flight-of-the-amazon-queen flightgear foliate fondo font-manager four-in-a-row free42 freedroid freenukum freeoffice frogatto-unstable frostwire-desktop FurnitureLibraryEditor gala ganttproject-bin geary gedit geogebra geogebra ghex ghidra giada gitkraken gitter gkrellm gnome-applets gnome-books gnome-boxes gnome-builder gnome-clocks gnome-color-manager gnome-connections gnome-control-center gnome-dictionary gnome-documents gnome-firmware-updater gnome-inform7-unstable gnome-klotski gnome-mahjongg gnome-mines gnome-multi-writer gnome-music gnome-network-displays gnome-nibbles gnome-notes gnome-packagekit gnome-passwordsafe gnome-photos gnome-podcasts gnome-power-manager gnome-recipes gnome-robots gnome-screenshot gnome-session gnome-shell gnome-shell-extension-EasyScreenCast-unstable gnome-shell-extension-night-theme-switcher gnome-shell-gsconnect gnome-shell-pomodoro gnome-shell-system-monitor-unstable gnome-software gnome-sound-recorder gnome-sudoku gnome-taquin gnome-terminal gnome-tetravex gnome-tour gnome-tweaks gnome-usage gnome-user-share gnujump goattracker goattracker-stereo goland golden-cheetah gomuks gpaste gpscorrelate gpsprune gradio groove gthumb gtk-gnutella gucharmap hakuneko hashit hdfview hexchat hitori hyperrogue iagno idea-community idea-ultimate ideamaker ideogram IPMIView irccloud ivan jabref jack1 jack2 jack2 jackmix jameica jd-gui jdiskreport jitsi jmol joplin-desktop jpexs keen4 keepass keeweb kgx-unstable khronos koboredux koboredux kodestudio kooha kupfer LazyLimiter lbry-desktop ledger-live-desktop lens leo-editor lighttable littlegptracker-unstable lollypop losslesscut lsp-plugins ltc-tools lunar-client luppp lure-of-the-temptress lutris lutris lyrebird maelstrom malcontent-ui mame mame mame marathon marathon-eternal marathon-evil marathon-infinity marathon-pheonix marathon-red mari0 marktext-v0.16.2-binary mate-applets mate-control-center mate-tweak MBdistortion mediainfo-gui meld mendeley metadata-cleaner meteo mgba michabo midori midori mikutter mimeo mindustry mindustry minecraft-launcher minetime mission-planner mlterm molotov monero-gui monitor mooSpace-unstable mousai mps mrrescue munt mutter mutter MyCrypto nautilus nautilus-python netbeans netlogo nixui notable nottetris2 nuclear-v0.6.6 numberstation obsidian octant-desktop octave octave octave oh-my-git open-ecard openboard OpenJK OpenTabletDriver opentrack orthorobot ostinato osu-lazer p3x-onenote palemoon pantheon-agent-geoclue2 Patchwork pathways-into-darkness pdfsam-basic peek perl5.32.1-frozen-bubble pharo pharo-launcher phoc phosh phpstorm pika-backup piper plater plexamp pluginUtils polari portfolio PortfolioPerformance postman pragha protonvpn-linux-gui prusa-slicer prusa-slicer pulseeffects pulseeffects pycharm-community pycharm-professional pymol PySolFC pyspread python3.8-spyder qpaeq quadrapassel quartus-prime-lite quilter rabbitvcs radicle-upstream rambox rambox-pro react-native-debugger recapp RhythmDelay ricochet rider rimshot ripcord robo3t robo3t rocksndiamonds RStudio RStudio rubicon-x ruby-mine runelite rxvt-unicode rxvt-unicode rxvt-unicode-unwrapped rxvt-unicode-unwrapped ryujinx sage sage sage saleae-logic scenebuilder scenic-view scid-vs-pc sdlpop sequeler setzer sfxr sgt-puzzles-r20200610.9aa7b7c shelfMultiBand shortwave shotcut shotwell sideload SideQuest sienna simplenote slic3r slimerjs smartgithg snapdragon-profiler-v2021.2 snippetpixie softmaker-office sonic-pi soulseekqt soundtracker sourcetrail spacefm spyder squirrel-sql standardnotes station stm32cubemx streamlink-twitch-gui-bin stretchly sublimetext subtitleeditor super-slicer svxlink sweethome3d-application swell-foop swingsane switchboard switchboard-plug-pantheon-shell switchboard-plug-power Sylk sysprof system-config-printer system-config-printer system-syzygy tali Tambura tangram teamspeak-client teleprompter tensor-git termius tetraproc TexturesLibraryEditor thunderbird tilix timeline timeular tla-toolbox toggldesktop tome2 tome4 tor-browser-bundle-bin tor-browser-bundle-bin transgui trezor-suite trilium-desktop tusk-v0.23.0 tuxguitar tvbrowser-bin unciv unityhub ut2004 vala-language-server vapor vice vieb viewnior vimb vimb vis visualvm vmware-view VoiceOfFaust vscode vscode-extension-ms-vsliveshare-vsliveshare vscode-with-extensions vscodium wasabiwallet wavebox wayst-unstable webbrowser webcamoid webstorm whirlpool-gui wingpanel wingpanel wingpanel-applications-menu wingpanel-indicator-bluetooth-unstable wingpanel-indicator-datetime wingpanel-indicator-keyboard wingpanel-indicator-network wingpanel-indicator-nightlight wingpanel-indicator-notifications wingpanel-indicator-power wingpanel-indicator-session-unstable wingpanel-indicator-sound wire-desktop wootility WorldOfGoo write_stylus xbill xdg-desktop-portal xdg-desktop-portal-gtk xmind xonotic-dedicated xonotic-glx xonotic-sdl xournal ytmdesktop zathura-with-plugins zettlr zotero zsnes zulip`

Reply:
I've rebased on staging. Do you know where the branches are documented?